### ISSUE TITLE
Example Swift code can not contain blank lines

### ISSUE THREAD

**Issue by blochberger** [realm/jazzy#628]
I documented example usage of a function by providing Swift code with

`````` swift
/**
    Does something.

    #### Example
    ```swift
    class SomethingSpecific: Something {
        typealias Error = String

        func doSomething() {
            …
        }
    }
    ```
*/
public class Something {
}
``````

And the output is something like

> Does something.
> 
> **Example**
> 
> ``` swift
> class SomethingSpecific: Something {
>   typealias Error = String
> 
> <span class="kd">func</span> <span class="nf">doSomething</span><span class="p">()</span> <span class="p">{</span>
>    <span class="err">…</span>
> <span class="p">}</span>
> 
> }
> ```

It works fine in the `README.md` file I use as the appendix.

**Comment by istx25** [comment]
Strange! Thanks for filing this issue. I'll do some digging and come back with what I find.

**Comment by jpsim** [comment]
Fixed in #766.
